spec/models/metric_spec.rb

```diff
@@ -531,7 +531,7 @@
                                                       :resource  => @host1,
                                                       :timestamp => "2010-04-14T22:00:00Z")
           metrics = Metric::Finders.find_all_by_day([@vm1, @host1], "2010-04-14T00:00:00Z", 'hourly', @time_profile)
-          expect(metrics.collect(&:resource_type).uniq.sort).to eq(%w(VmOrTemplate Host).uniq.sort)
+          expect(metrics.collect(&:resource_type).uniq).to match_array(%w(VmOrTemplate Host).uniq)
```

Comment:
Minor, but the latter uniq is not necessary.  Don't bother changing unless you need to push a new commit.